### DISCUSSION TITLE
https://material-web.dev/

### DISCUSSION THREAD

**Discussion by MarkStega** [material-components/material-web#4790] [Q&A] [answered]
Is the source for [https://material-web.dev/](https://material-web.dev/) available in a public repository? Although each component that is documented has good example code, I'd really like to see how the landing page & menu of that website is constructed.

**Comment by asyncliz** [comment, marked as answer]
Yup, right here! https://github.com/material-components/material-web/tree/main/catalog

**Reply by omar2205** [reply]
Sorry for asking, but why not put it on `material.io`? like `web.material.io`?

**Reply by asyncliz** [reply]
At the moment it's easier for us to develop and update the catalog off of mio

**Reply by omar2205** [reply]
ok thanks, have a nice day :)
